README.md

```diff
@@ -8,14 +8,43 @@ ar-gif is an effort to bring augmented reality with web components in a easy way
 This web-component is used in [Jandig](https://github.com/memeLab/Jandig), a project the aims to bring AR and Art for everyone.
 
 ### Usage
-ar-gif has a simple API, we have an ar-scene and one or more ar-markers. 
+ar-gif has a simple API, we have an ar-scene and one or more ar-markers.
 ```html
 <ar-scene>
   <ar-marker patt="hiro.patt" content="hiro.gif"></ar-marker>
-  <ar-marker patt="cat.patt" content="cat.mp4"></ar-marker>  
+  <ar-marker patt="cat.patt" content="cat.mp4"></ar-marker>
+</ar-scene>
+```
+
+You can use also use some simple attributes to control the appeareance.
+
+- scale : as a two dimensional vector
+- position : as a three dimensional vector
+- rotation : as a three dimensional vector
+- audio : [0|1] If your mp4 supports audio. The default behavior is muted playback.
+- loop : [0|1] Stop a mp4 video at the end. The default behavior is looped playback.
+
+
+```html
+<ar-scene>
+  <ar-marker patt="hiro.patt" content="hiro.gif" scale="1 1" position="0 0 0" rotation="0 0 0"></ar-marker>
+  <ar-marker patt="cat.patt" content="cat.mp4" audio="1" loop="0"></ar-marker>
```

Comment:
For a more semantic style, I think "0" or "1" could be changed to "true" or "false".